src/batou_ext/resources/modsecurity-configuration.nix

```diff
@@ -0,0 +1,73 @@
+{ ... }:
+{
+  environment.etc."modsecurity_{{component.ruleset_name}}.conf".text = ''
```

Comment:
lets make a single file in /etc/nginx/modsecurity/... or so to avoid clobbering /etc.